commit.go

```diff
@@ -166,6 +191,17 @@ func (b *Builder) Commit(ctx context.Context, dest types.ImageReference, options
 			logrus.Debugf("error destroying signature policy context: %v", err2)
 		}
 	}()
+
+	// Check if we'd be allowed to run an image from the destination location.
+	allowed, err := policyContext.IsRunningImageAllowed(ctx, &referenceOnlyImage{dest: dest})
+	if err != nil {
+		return "", nil, "", errors.Wrapf(err, "running image with reference %q would not be allowed by policy, denying commit", transports.ImageName(dest))
```

Comment:
This wrap should be error checkinf if running image with reference %q would not be allowed...

Reply:
An earlier revision did this, but we get a non-nil `err` back when `allowed` is false to explain the reason why `allowed` is false (an [example](https://github.com/containers/image/blob/dd0113663d0da85521e6ead3ea4ec45b050775f1/signature/policy_eval_simple.go#L28), and that phrasing made the error message read confusingly.